integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/EDUCoreConverter.java

```diff
@@ -39,10 +40,12 @@
         try {
             Marshaller marshaller = jaxbContext.createMarshaller();
             marshaller.marshal(new JAXBElement<>(new QName("uri", "local"), EDUCore.class, message), os);
-            message.setPayload(payloadConverter.unmarshallFrom(payloadAsString.getBytes()));
+            message.setPayload(payloadConverter.unmarshallFrom(payloadAsString.getBytes("UTF-8")));
             return os.toByteArray();
         } catch (JAXBException e) {
             throw new RuntimeException("Unable to create marshaller for " + EDUCore.class, e);
+        } catch (UnsupportedEncodingException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Combine this catch with the one at line 45, which has the same body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2147)